app/src/main/res/raw/sessions_ko.json

```diff
@@ -671,8 +671,8 @@
       "name": "Room C"
     },
     "language_id": "ja",
-    "slide_url": "",
-    "movie_url": "",
+    "slide_url": "https://speakerdeck.com/masaki_mori/view-into-the-abyss",
+    "movie_url": "https://www.youtube.com/watch?v=cLjbUqmyop0",
```

Comment:
This is where I missed to add the `movie_url` 🙇  I happened to find this mistake when I was adding `slide_url` so I added the missing `movie_url` as well.